storage/container_storage_interface/persistent-storage-csi-manila.adoc

```diff
@@ -0,0 +1,24 @@
+[id="persistent-storage-csi-manila"]
+= OpenStack Manila CSI Driver Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-manila
+
+toc::[]
+
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for the link:https://wiki.openstack.org/wiki/Manila[OpenStack Manila] shared file system service.
+
+Familiarity with link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/[PVs], link:https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims[persistent volume claims (PVCs)], link:http://blog.kubernetes.io/2016/10/dynamic-provisioning-and-storage-in-kubernetes.html[dynamic
+provisioning], and link:https://kubernetes.io/docs/admin/authorization/rbac/[RBAC authorization] is recommended.
+
+Before PVCs can be created, you must install the Manila CSI Driver Operator. The Operator creates the ManilaDriver Custom Resource that is required in the {product-title} cluster. It also creates the required storage classes for all available Manila share types needed for dynamic provisioning.
```

Comment:
The Operator doesn't create the ManilaDriver Custom Resource. It should be created by the cluster administrator manually after the installation from this manifest: https://github.com/openshift/csi-driver-manila-operator/blob/master/deploy/crds/csi.openshift.io_v1alpha1_maniladriver_cr.yaml

Reply:
Good catch. I've updated to mention that the driver must be installed manually and linked to the related set of instructions that come later on the page.